covid/data.py

```diff
@@ -80,7 +80,8 @@ def process_covidtracking_data(data: pd.DataFrame, run_date: pd.Timestamp):
         :,
     ] = 0
 
-    return data
+    # the right bound of the slice has a -1 day, because the diff operation adds a day to the end
+    return data.loc[idx[:, :(run_date - pd.DateOffset(1))], ["positive", "total"]]
```

Comment:
Not sure I follow:
```
In [3]: pd.Series([1, 2, 3]).diff()
Out[3]:
0    NaN
1    1.0
2    1.0
```
The right-most item is unaffected, so why do we need to remove one day here?

Reply:
I don't remember exactly why I did this (a few days ago).
The PR should not change anything about the resulting DataFrame

Reply:
I don't see how its not changing that, and we don't have tests for this I think. If it does probably requires a better comment.

Reply:
Yes. I will take a close look after lunch. Will ping you when ready